src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/addressing.html.twig

```diff
@@ -31,3 +31,38 @@
         </div>
     </div>
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
```

Comment:
We definitely need to think about nice way for including page specific JS. I am wondering if putting this in app.js is not a better option, to keep the templates clean.